example/server/integrations.py

```diff
@@ -115,10 +114,9 @@ def get_stellar_toml():
         },
         "CURRENCIES": [
             {
-                "code": asset.code,
-                "issuer": polaris.settings.STELLAR_ISSUER_ACCOUNT_ADDRESS,
+                "code": "USD",
+                "issuer": "GBHJFK4LOLMYV34P25WBKDPVX75ANB3QQJCRBYO4MGWGFTUE7DBBW2PI",
```

Comment:
why are we hardcoding this here?

Reply:
I need to change it back so good catch. I hard coded because locally, `example` uses the `polaris` source code, which no longer used `STELLAR_ISSUER_ACCOUNT_ADDRESS`.